checador/app/assets/stylesheets/records.scss

```diff
@@ -0,0 +1,13 @@
+.box{
+  margin: 0 auto;
```

Comment:
I recommend you to order all the properties in alphabetical order, what do you think?

Reply:
Yeah, i think it's a good practice